Sources/_FoundationInternals/String/String+Internals.swift

```diff
@@ -16,4 +16,30 @@ extension String {
             $0.properties.isWhitespace
         })
     }
+
+    static func _tryFromUTF16(_ input: UnsafeBufferPointer<UInt16>) -> String? {
```

Comment:
Nitpick about the name. `try` means something specific in Swift, so I feel we should just name this something like `init?(fromUTF16:)` if possible.

Reply:
Agreed, but sounds like we're going for parity with the already extant `_tryFromUTF8` from the stdlib?

Reply:
I think that's it, yes. If we were to make it an `internal init` instead, it should have an underscored argument label.